stixpy/science.py

```diff
@@ -243,74 +251,365 @@ def plot_pixels(self, time_indices=None, energy_indices=None, fig=None):
+            cbar = plt.colorbar(cm.ScalarMappable(norm=norm, cmap=clrmap),
+                                orientation='vertical', cax=cax)
+            cbar.ax.set_title(f'{str(counts.unit)}', rotation=90,
+                              x=-0.8, y=0.4)
+
+        def instrument_layout(fig, font):
+            """ Shows the layout of the instrument to make it easier to locate the detectors. """
+            x = [0, 2]
+            y = [1, 1]
+            fig.add_axes([0.06, 0.055, 0.97, 0.97])
+            plt.plot(x, y, c='b')
+            plt.plot(y, x, c='b')
+            plt.axis('off')
+            fig.add_axes([0.09, 0.08, 0.91, 0.92])
+            draw_circle_1 = plt.Circle((0.545, 0.540), 0.443, color='b',
+                                       alpha=0.1)
+            draw_circle_2 = plt.Circle((0.545, 0.540), 0.07, color='#2b330b',
+                                       alpha=0.95)
+            fig.add_artist(draw_circle_1)
+            fig.add_artist(draw_circle_2)
+            plt.axis('off')
+
+            # Label the quandrants of the instrument
+            fig.add_axes([0, 0, 1, 1])
+            plt.text(0.19, 0.89, 'Q1', **font)
+            plt.text(0.19, 0.17, 'Q2', **font)
+            plt.text(0.86, 0.17, 'Q3', **font)
+            plt.text(0.86, 0.89, 'Q4', **font)
+            plt.axis('off')
+
+        instrument_layout(fig, quadrant_font)  # Call the instrument layout
+
+        # Create the energy and time slider add the bottom of the figure
         axcolor = 'lightgoldenrodyellow'
         axenergy = plt.axes([0.15, 0.05, 0.55, 0.03], facecolor=axcolor)
-        senergy = SliderCustomValue(axenergy, 'Energy', 0, len(energies)-1, format_func=energyval,
+        senergy = SliderCustomValue(ax=axenergy,
+                                    label='Energy',
+                                    valmin=0,
+                                    valmax=len(energies)-1,
+                                    format_func=energyval,
                                     valinit=0, valstep=1)
         axetime = plt.axes([0.15, 0.01, 0.55, 0.03], facecolor=axcolor)
-        stime = SliderCustomValue(axetime, 'Time', 0, counts.shape[0]-1, format_func=timeval,
-                                  valinit=0, valstep=1)
+        stime = SliderCustomValue(ax=axetime,
+                                  label='Time',
+                                  valmin=0,
+                                  valmax=counts.shape[0]-1,
+                                  format_func=timeval,
+                                  valinit=1, valstep=1)
 
         pixel_ids = [slice(0, 4), slice(4, 8), slice(8, 12)]
         if counts.shape[2] == 4:
             pixel_ids = [slice(0, 4)]
 
         containers = defaultdict(list)
 
+        xnorm = plt.Normalize(SubCollimatorConfig["SC Xcen"].min()*1.5, SubCollimatorConfig["SC Xcen"].max()*1.5)
+        ynorm = plt.Normalize(SubCollimatorConfig["SC Ycen"].min()*1.4, SubCollimatorConfig["SC Ycen"].max()*1.4)
```

Comment:
we scale the position X with 1.5 to avoid to much margins.
the 1.4 for y is because we need a bit more margin in between for the detector name: 

![image](https://user-images.githubusercontent.com/2226714/124491727-e07b6600-ddb3-11eb-85b3-2a123b77748d.png)